tools/wptrunner/wptrunner/testloader.py

```diff
@@ -51,6 +51,12 @@ def __contains__(self, key):
     def __getitem__(self, key):
         return self._data[key]
 
+def update_include_from_files(file):
+    new_include = []
+    with open(file) as f:
+        for line in f:
+            new_include.append(line.rstrip())
+    return new_include
```

Comment:
Can you also toss a simple test in /tools/wptrunner/wptrunner/tests/test_testloader.py?